modules/ide-integration/README.md

```diff
@@ -1,228 +1,292 @@
-  ```
+## 🖥️ Starting Airavata Components
 
-* Wait until all the services come up. This will initialize all utilities required to start Airavata server
+### 6️⃣ Start API Server
 
-### Starting API Server
+1. **Navigate to:** `org.apache.airavata.ide.integration.APIServerStarter`
+2. **Right-click** in the editor
+3. **Select:** `Run 'APIServerStarter.main()'`
 
-#### Note: For JDK 11+
-you have to add ``--add-opens java.base/java.lang=ALL-UNNAMED`` as a JVM argument
+> 💡 **JDK 17+ Note:** Add this JVM argument in your run configuration:
+> ```
+> --add-opens java.base/java.lang=ALL-UNNAMED
+> ```
 
-* Go to org.apache.airavata.ide.integration.APIServerStarter class and right click on the editor and click Run option. This will start Airavata server
+### 7️⃣ Start Job Execution Engine
 
-### Starting Job Execution Engine
+1. **Navigate to:** `org.apache.airavata.ide.integration.JobEngineStarter`
+2. **Right-click** and select **Run**
 
-* Go to org.apache.airavata.ide.integration.JobEngineStarter class and right click on the editor and click Run option. 
-This will start all components of Job Execution Engine including Helix Controller, Helix Participant, Pre Workflow Manager and 
-Post Workflow Manager
+**Components Started:**
+- 🔄 Helix Controller
+- 👥 Helix Participant  
+- ⚙️ Pre Workflow Manager
+- 📋 Post Workflow Manager
 
-### Starting Job Monitoring components
+### 8️⃣ Start Job Monitoring
 
-* This will start the Email Based Job Monitoring agent. Before starting this, you have to create a new gmail account by going to 
-https://accounts.google.com/signup
+**Setup Email Monitor (One-time setup):**
 
-* Once the account is created, turn on 2-Step Verification and create an App Password (Use the type "Other" from the App type selection and give the name as "Airavata")
-https://myaccount.google.com/security
+1. **Create Gmail Account:** [accounts.google.com/signup](https://accounts.google.com/signup)
+2. **Enable 2-Step Verification:** [myaccount.google.com/security](https://myaccount.google.com/security)
+3. **Generate App Password:**
+   - Type: "Other" 
+   - Name: "Airavata"
 
-* Update the email address and App Password in src/main/resources/airavata-server.properties file
+4. **Update Configuration:**
+   Edit `src/main/resources/airavata-server.properties`:
+   ```properties
+   email.based.monitor.address=your-email@gmail.com
+   email.based.monitor.password=your-app-password
+   ```
 
-  email.based.monitor.address=CHANGEME
-  email.based.monitor.password=CHANGEME
-  
-* Go to org.apache.airavata.ide.integration.JobMonitorStarter class and right click on the editor and click Run option.
+5. **Start Monitor:**
+   - Navigate to: `org.apache.airavata.ide.integration.JobMonitorStarter`
+   - Right-click and **Run**
 
-### Starting User Portal (Django Portal)
+## 🌐 User Portal Setup (Django)
 
-* You can create and launch experiments and manage credentials using this portal
```

Comment:
I believe we should keep this line so the user knows what this Django portal is supposed to do